avm/res/api-management/service/diagnostic/main.bicep

```
@@ -0,0 +1,93 @@
+metadata name = 'API Management Service Diagnostics'
+metadata description = 'This module deploys an API Management Service Diagnostic.'
+
+@description('Conditional. The name of the parent API Management service. Required if the template is used in a standalone deployment.')
+param apiManagementServiceName string
+
+@description('Required. Diagnostic Name.')
+param name string
+
+@description('Required. Logger resource ID.')
+param loggerId string
```

```suggestion
param loggerResourceId string
```
Note: This has implicatinos for other places too